### DISCUSSION TITLE
Is it have some drone control relate projects using MicroPython?

### DISCUSSION THREAD

**Discussion by water5** [micropython/micropython#10560] [General]
Is it have some drone control relate projects using MicroPython?

**Comment by peterhinch** [comment]
I have not heard of anyone doing this, but it should certainly be possible.

**Comment by cibernakaq** [comment]
There is a recent development, the Scout Flight Controller.  This is the [repo](https://github.com/TimHanewich/scout/tree/master).

**Reply by peterhinch** [reply]
Very impressive!

**Comment by sophiedegran** [comment]
Yes he have made a Quadcopter under raspberry pi pico! He is on medium to timhanewich.medium.com ☺

**Comment by mattytrentini** [comment]
Tim's 12-part write-up about development on building Scout is _excellent_, highly recommended! He's also agreed to talk about Scout at the [November Melbourne MicroPython Meetup](https://www.meetup.com/micropython-meetup/events/297139462/) (I haven't yet updated the event to list the agenda). As usual, we'll live stream and record the talk.
